generated/nidigital/nidigital/session.py

```diff
@@ -1134,7 +1566,7 @@ def create_source_waveform_serial(self, waveform_name, data_mapping, sample_widt
     def disable_sites(self):
         r'''disable_sites
 
-        TBD
+        Disables specified sites. Disabled sites are not included in pattern bursts initiated by the _initiate method or the _burst_pattern method, even if the site is specified in the list of pattern burst sites in configure_pattern_burst_sites method or in the **siteList** input of the _burst_pattern method. Additionally, if you specify a list of pin or pin group names in a **channelList** parameter in any NI-Digital method, digital pattern instrument channels mapped to disabled sites are not affected by the method. The methods that return per-pin data, such as the ppmu_measure method, do not return data for channels mapped to disabled sites. The digital pattern instrument channels mapped to the sites specified are left in their current state. NI TestStand Semiconductor Module requires all sites to always be enabled, and manages the set of active sites without disabling the sites in the digital instrument session. Do not use this method with the Semiconductor Module.
```

Comment:
".. or in the **siteList** input of the _burst_pattern method"

**siteList** input --> sites repeated capability attribute

Reply:
Referred to "repeated capabilities".